homeworks/react-basics/yahello_yana/kottans-lectures/public/data.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
You don't need this file, you can get data from ```https://raw.githubusercontent.com/kottans/chernivtsi-frontend-2019/master/homeworks/react-basics/data.json```

Reply:
Thank you, fixed!